private/bufpkg/bufplugin/bufplugindocker/docker.go

```diff
@@ -215,6 +224,20 @@ func NewClient(logger *zap.Logger) (Client, error) {
 	}, nil
 }
 
+type ClientOption func(options []client.Opt) []client.Opt
+
+func WithHost(host string) ClientOption {
```

Comment:
In order to test against a httptest.Server Docker engine, needed to expose this option for overriding the default host. Alternatively, I could use `DOCKER_HOST` env var but I didn't think that would work reliably with parallel tests.

Reply:
Sounds good, let's leave a comment explaining that. It's OK to keep this around for testing as long as it's clear.